modules/distribution/product/src/main/resources/conf/templates/repository/conf/registry.xml.j2

```diff
@@ -124,6 +124,27 @@
          </filter>
     </handler>
 
+    <!--Config for custom registry handlers.-->
+    {% for handler in registry_handler %}
+    <handler class="{{handler.class}}" {% if handler.method is defined %} method="{{handler.method}}" {% endif %}>
+        {% for propKey,propValue in handler.properties.items() %}
+        <property name="{{propKey}}">{{propValue}}</property>
+        {% endfor %}
+        {% if handler.nested_property.enable %}
+        <property name="{{handler.nested_property.name}}" type="{{handler.nested_property.type}}">
+            {% for prop in nested_properties[handler.nested_property.name] %}
+                <{{prop.tag}} {% if prop.key is defined %} key="{{prop.key}}" {% endif %}>{{prop.value}}</{{prop.tag}}>
+            {% endfor %}
+        </property>
+        {% endif %}
+        <filter class="{{handler.filter_class}}">
+            {% for filterPropKey,filterPropValue in handler.filter_properties.items() %}
+            <property name="{{filterPropKey}}">{{filterPropValue}}</property>

```

Comment:
can we give https://github.com/wso2/product-apim/pull/9982/files#diff-96595b13a206dbf1cac03e6a8929c3a64c212c083fe0e6a7d96e4339cb7131e1R135-R136 here as well